ts-apps/decision-logic-visualizer/src/lib/data/viz-expr-to-lir.ts

```diff
@@ -70,11 +75,18 @@ export const LadderGraphLirSource: LirSource<IRExpr, LadderGraphLirNode> = {
 // TODO2: Attach a group id to the label for the flownode to make it easier to debug
 // TODO3: Return the number of groups as metadata in the FlowGraph
 
+export interface ToLirResult {
+  graph: DirectedAcyclicGraph<LirId>
+  vizExprToLir: Map<IRId, DirectedAcyclicGraph<LirId>>
```

Comment:
Ignore the threading of the `vizExprToLir`: it only really gets used in the next PR (and there's a bug in the traversal / threading that's fixed in the next PR).